api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedSiteRouter.java

```diff
@@ -157,4 +155,20 @@ private void handleDeleteSiteRoute(RoutingContext ctx) {
 
     end(ctx.response(), 200);
   }
+
+  private void registerEditSite(Router router) {
+    Route editSiteRoute = router.post("/:site_id/edit");
```

Comment:
❓ Can you add this new route to the dev docs?

here's the file to change: https://github.com/Code-4-Community/c4c-dev-docs/blob/master/docs/sftt/sftt-api-spec.md

Reply:
I just made a PR!

https://github.com/Code-4-Community/c4c-dev-docs/pull/78